portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -231,6 +231,50 @@ definition {
 		}
 	}
 
+	@description = "LPS-117341 Create a label for one of the categories and add another one with the text "..." and tooltip or popover with all the categories left"
+	@priority = "5"
+	test CheckForMoreCategories {
+		property portal.acceptance = "true";
+
+		Category.openCategoriesAdmin(siteURLKey = "global");
+
+		task ("Create a list of Audiences") {
+			for (var count : list "1,2,3") {
+				ContentDashboard.addAudience(title = "teste ${count}");
+			}
+		}
+
+		task ("Navigate to Web content Created and add of audiences") {
+			JSONWebcontent.addWebContent(
+				content = "test content",
+				groupName = "Test Site Name",
+				title = "WC");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContentNavigator.gotoEditCP(webContentTitle = "WC");
+
+			for (var count : list "1,2,3") {
+				WebContent.selectCategory(
+					categoryName = "teste ${count}",
+					vocabularyName = "Audience");
+			}
+
+			PortletEntry.publish();
+		}
```

Comment:
This is not needed, you can use JSON macro to create a web content with several categories.